sw/banshee/src/tran.rs

```diff
@@ -23,6 +23,7 @@ static NONAME: &'static i8 = unsafe { std::mem::transmute("\0".as_ptr()) };
 
 /// Base address of the stream semantic regsiters
 static SSR_BASE: u64 = 0x204800;
+static SSR_N_STREAMERS: u32 = 2;
```

Comment:
Awesome. If you add a new variable for the number of streamers, can I ask you to also update [line 2753](https://github.com/pulp-platform/snitch/blob/b72c089b206785ae0b7ca331d104f52765e134c1/sw/banshee/src/tran.rs#L2753), which seems to have the number of streamers hardcoded to 2. Or am I misunderstanding something?

Reply:
That is correct! I looked for other ocurences where `SSR_N_STREAMERS` can be substituted but there might still be some missing though.